cs/cli/vw_example.cpp

```diff
@@ -2,6 +2,8 @@
 // individual contributors. All rights reserved. Released under a BSD (revised)
 // license as described in the file LICENSE.
 
+#define NOMINMAX
```

Comment:
this used to be inside a ifdef _WIN32, is there a reason to not follow that

Reply:
CLI will only build on windows as far as I know so I figured it wasnt necessary